signs-work-order-map/config/map.ts

```diff
@@ -0,0 +1,127 @@
+import "mapbox-gl/dist/mapbox-gl.css";
+// import { SymbolLayerSpecification, RasterLayerSpecification } from "mapbox-gl";
+
+// The Nearmap API key is managed by CTM. Contact help desk for maintenance and troubleshooting.
+const NEARMAP_KEY = process.env.NEXT_PUBLIC_NEARMAP_KEY;
```

Comment:
You may still plan on incorporating it, and if so, please ignore this, but this key doesn't look like it's used anywhere.